schema/spacewalk/susemanager-schema.spec

```diff
@@ -111,6 +118,13 @@ systemctl is-active --quiet uyuni-check-database.service && {
 systemctl try-restart uyuni-check-database.service ||:
 
 %files
+%if 0%{?suse_version}
+%dir /usr/share/susemanager
+/usr/share/susemanager/update-messages.txt
+%ghost /var/adm/update-messages/%{name}-%{version}-%{release}
+%endif
```

Comment:
I think the main package should have the schema files. Seems they still belong to utility sub-package